spec/models/education_benefits_claim_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe EducationBenefitsClaim, type: :model do
+  let(:attributes) do
+    {
+      form: { foo: true }
+    }
+  end
+  subject { described_class.new(attributes) }
+
+  describe "validations" do
+    it { should validate_presence_of(:form) }
```

Comment:
you could initialize subject using factory girl build, then specify you want form: nil, that would have the same effect I would think.